src/m365/spo/commands/site/site-chrome-set.ts

```diff
@@ -0,0 +1,179 @@
+import { Logger } from '../../../../cli';
+import {
+  CommandOption
+} from '../../../../Command';
+import GlobalOptions from '../../../../GlobalOptions';
+import request from '../../../../request';
+import SpoCommand from '../../../base/SpoCommand';
+import commands from '../../commands';
+
+export enum HeaderLayout {
+  Standard = 1,
+  Compact,
+  Minimal,
+  Extended
+}
+
+export enum FooterLayout {
+  Simple = 1,
+  Extended
+}
+
+export enum Alignment {
+  Left = 0,
+  Center,
+  Right
+}
+
+export enum Emphasis {
+  Lightest = 0,
+  Light,
+  Dark,
+  Darkest
+}
+
+interface CommandArgs {
+  options: Options;
+}
+
+interface Options extends GlobalOptions {
+  url: string;
+
+  headerLayout?: HeaderLayout;
+  headerEmphasis?: Emphasis;
+  logoAlignment?: Alignment;
+  footerLayout?: FooterLayout;
+  footerEmphasis?: Emphasis;
+  disableMegaMenu?: boolean;
+  hideTitleInHeader?: boolean;
+  disableFooter?: boolean;
+}
+
+class SpoSiteChromeSet extends SpoCommand {
+
+  public get name(): string {
+    return commands.SITE_CHROME_SET;
+  }
+
+  public get description(): string {
+    return 'Set the chrome header and footer for the specified site';
+  }
+
+  public getTelemetryProperties(args: CommandArgs): any {
+    const telemetryProps: any = super.getTelemetryProperties(args);
+    telemetryProps.headerLayout = typeof args.options.headerLayout !== 'undefined';
```

Comment:
Since these values don't carry PII, it would be better to track the actual values so that we can understand what values are being set

Reply:
Updated